erp/export/export.py

```diff
@@ -0,0 +1,21 @@
+import csv
+import json
+from dataclasses import asdict
+from typing import List
+
+
+def export_to_csv(file, headers, mapped_data: List):
+    csv_writer = csv.DictWriter(file, fieldnames=headers)
+    csv_writer.writeheader()
+
+    for erp in mapped_data:
+        data = asdict(erp)
+        # TODO: better to be in dict_factory
+        # Solve the quoting problem: frictionless only validate with "[""value""]",
```

Comment:
I highly doubt the JSON string escaping of array values is natively handled by Excel, as Excel doesn't natively deal with arrays for cell values. It's not because Python's DictWriter claims they support an "Excel" setting that it will magically implement and support this kind of frictionless/tableschema specificity… At least we should definitely test what's the contributor experience is when crafting a file in Excel and trying to import it before publishing the schema.

Reply:
I'm just using JSON to get the correct ouput (double quotes surrounding and double quotes escaped/doubled inside). I've just checked with other schemas on ETALAB, and they also do like this.

Reply:
Haha OK if Etalab is okay with it we should probably go with it 👍